utils/src/main/scala/za/co/absa/enceladus/utils/validation/ValidationLevel.scala

```diff
@@ -20,5 +20,10 @@ object ValidationLevel extends Enumeration {
 
   final val NoValidation, ForRun, Strictest = Value
 
-  final val NoValidationName = "NoValidation" // This is used in an annotation - for that the type MUST NOT be specified
+  object Constants { // Not to mix with enumeration items
+    final val DefaultValidationLevel:ValidationLevel  = NoValidation
```

Comment:
```suggestion
    final val DefaultValidationLevel: ValidationLevel = NoValidation
```

Also, I keep wondering if the existence of `DefaultValidationLevel` is necessary, but sure, why not.

Reply:
I think it does. It prevents accidental removal/change without realizing the consequences - at least I believe it does.